bundles/org.openhab.binding.pushover/src/main/java/org/openhab/binding/pushover/internal/connection/PushoverMessageBuilder.java

```diff
@@ -279,27 +283,23 @@ public ContentProvider build() throws PushoverCommunicationException {
         return body;
     }
 
-    private Path createTempFile(byte[] data) throws PushoverCommunicationException {
+    private Path createTempFile(byte[] data) throws CommunicationException {
         try {
             Path tmpFile = Files.createTempFile("pushover-", ".tmp");
             return Files.write(tmpFile, data);
         } catch (IOException e) {
-            logger.debug("IOException occurred while creating temp file - skip sending message: {}",
-                    e.getLocalizedMessage(), e);
-            throw new PushoverCommunicationException(
-                    String.format("Skip sending the message: %s", e.getLocalizedMessage()), e);
+            logger.debug("IOException occurred while creating temp file - skip sending message: {}", e.getMessage(), e);
+            throw new CommunicationException(String.format("Skip sending the message: %s", e.getMessage()), e);
```

Comment:
You could have created a `@text` entry for this message too. But you decide.

Reply:
Good catch. Thanks for the hint.